es/resources/community.md

```diff
@@ -7,15 +7,80 @@ lang: es
 
 # Comunidad
 
+## Comité Técnico
+
+El comité técnico de Express se reúne en línea cada dos semanas (según sea necesario) para discutir el desarrollo y el mantenimiento de Express, entre otras incidencias relevantes para el proyecto Express. Cada reunión se anuncia normalmente en una [incidencia de expressjs/discussions](https://github.com/expressjs/discussions/issues) con un enlace para unirse o ver la reunión, que está abierta a todos los observadores.
```

Comment:
you could leave the term 'issue' untranslated, I feel that it would change the context of the text, or you could use 'problema' instead of 'incidencia'.

Reply:
I always thought that, you shouldn't translate terms that don't have a translation in Spanish and when they are translated they read differently in context, ok that's perfect, no problem.